features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_deleted.feature

```diff
@@ -0,0 +1,11 @@
+Feature: when the previous and current branches are deleted during a Git Town command, the main branch becomes the new previous branch
```

Comment:
[Previous discussion](https://github.com/Originate/git-town/pull/501#discussion_r29312041).

I don't feel strongly about it, so if you still feel this wording is better I'll reword the features.